Uno.Gallery/Uno.Gallery.Shared/Views/SamplePages/FlyoutSamplePage.xaml

```diff
@@ -164,72 +188,97 @@
 								</Button.Flyout>
 							</Button>
 						</smtx:XamlDisplay>
-						<smtx:XamlDisplay UniqueKey="FlyoutSamplePage_Fluent_Flyout_Icons" smtx:XamlDisplayExtensions.Header="MenuFlyout &amp; Icons">
+						<smtx:XamlDisplay UniqueKey="FlyoutSamplePage_Fluent_Flyout_Icons"
+										  smtx:XamlDisplayExtensions.Header="MenuFlyout &amp; Icons">
 							<Button Content="Open">
 								<Button.Flyout>
-									<MenuFlyout>
-
-										<MenuFlyoutItem Text="Like">
+									<MenuFlyout MenuFlyoutPresenterStyle="{StaticResource FluentMenuFlyoutPresenterStyle}">
+										<MenuFlyoutItem Text="Like"
+														Style="{StaticResource FluentMenuFlyoutItemStyle}">
```

Comment:
the xaml here serves not only as a showcase for the viewers, but also as reference source for copy-pasting
FluentMenuFlyoutPresenterStyle FluentMenuFlyoutItemStyle
these are not fluent style keys, that should be shown to the viewers...

Reply:
drop the x:Key from resources above, and remove the style property here
and see if the workaround still work

Reply:
@Xiaoy312 I tried removing the x:Key and styles as you suggested, but the workaround no longer seems to work 🤔
![image](https://github.com/unoplatform/Uno.Gallery/assets/78549750/80a627c4-55ae-4a26-ab28-631a35531646)
 I am not sure if there is any other way. Not ideal solution, but for the display we could just add a comment without the styles in the code

Reply:
Resolved internally with Xiao. The changes were not pushed to this PR

Reply:
> Resolved internally with Xiao. The changes were not pushed to this PR

For clarity, there is currently issue resolving resources while inside a flyout/popup. As the resources resolution is done by crawling up the visual tree and not the logical tree, the layers under the flyout-presenter (or a few layers above that, but before reaching its host, usually the `Button`) will not be able to resolve resources defined between the host `Button` and the xaml-root. 
Here is a visual-tree to illustrate the problem: // might not be exact, but you get the idea
```
win resolution hierarchy: B5-B4-(?:B3-B2)-A3-A2-A1-0
uno resolution hierarchy: B5-B4-B3-B2-B1-0

0 XamlRoot
	A1 Grid
		A2 MainPage
			A3 Button
	B1 PopupRoot
		B2 FlyoutPanel
			B3 MenuFlyoutPresenter
				B4 Border
					B5 Stuffs...
```